src/libs/Permissions.js

```diff
@@ -9,12 +11,14 @@ Onyx.connect({
     callback: val => betas = val || [],
 });
 
+const isDevelopment = lodashGet(Config, 'ENVIRONMENT', CONST.ENVIRONMENT.DEV) === CONST.ENVIRONMENT.DEV;
```

Comment:
I was thinking you should use `getEnvironment()` from the `Environment` module so that it will be cross-platform. However, that will also force this code to be async, which will be a pain to refactor. I believe it's only necessary to use `getEnvironement()` if you care about the difference between production and staging. Since you don't, it might be good to add a method to the `Environment` module called `isDevelopment()` which is a synchronous function (and then add method docs to explain why it's OK for it to be synchronous when `getEnvironment()` has to be async).

Reply:
Interesting. Why is that method returning a `Promise`? 

Could we alternatively just use the `__DEV__` global? It is being set here...

https://github.com/Expensify/Expensify.cash/blob/9a9099592330bc26d06408f34d76128718196eb4/config/webpack/webpack.dev.js#L37-L37

Reply:
> Interesting. Why is that method returning a Promise?

Why indeed, given it is getting a value from an already loaded object. Any objections to change that?

Reply:
The mobile versions of the environment module need to make a network request to get the info about if it's in staging or production. That's why it needs to be async.

Reply:
Oh I see now, because the only the web app uses the staging env configs and the native apps always use production config, but we need to know if they are TestFlight or google play beta versions.

Reply:
Oh, true, did not notice that. So what is preferred to add a isDevelopment that's not async or use the `__DEV__` global? If the latter, not sure how I grab that constant.

Reply:
I feel they are different concepts. It just happens that we are running in `__DEV__` mode when we are using the development environment. But we could also be running the development environment (i.e. not staging or production) and not be "running in dev mode".

Regardless, I'd think we want the environment in this case since whether you are running in dev mode or not the betas would probably be enabled in the "development environment".

Reply:
What about solving this from the API side? If PHP sends `all` in the beta information, then we should be fine. And PHP knows whether we are in DEBUG mode or not.

https://github.com/Expensify/Expensify.cash/blob/e28b50e139c492442ea984c3d48e23be744ceb4e/src/libs/Permissions.js#L17

Reply:
It is A LOT harder to detect where the request is coming from the server... keep in mind this has to work for contributors that do not point the API to the dev environment, since they don't have one.